docs/source/user-guide/tuning.md

```diff
@@ -37,6 +37,15 @@ Comet will allocate at least `spark.comet.memory.overhead.min` memory.
 
 If both `spark.comet.memoryOverhead` and `spark.comet.memory.overhead.factor` are set, the former will be used.
 
+## Memory Tuning in clusters
+In clusters like Kubernetes or YARN to make the resource manager respect correctly Comet memory parameters `spark.comet.memory*` 
+it is needed to pass to the starting command line additional Spark configuration parameter `--conf spark.plugins=org.apache.spark.CometPlugin`
```

Comment:
I'm curious why we don't specify the plugin in all cases instead of `--conf spark.sql.extensions=org.apache.comet.CometSparkSessionExtensions` since the plugin does (or can) register the session extensions?

Other Spark accelerators specify a plugin.

Reply:
Thats a great question. Its not needed for local run, but Spark in enterprises get started mostly in clusters. Perhaps we can also update the installation guide and make a remark about the plugin if people run the Comet in cluster mode

Reply:
I think it is mostly because we develop the extension at the beginning of Comet project. So all documents and customs are to specify the extension. We develop the Comet plugin after a while when we want to have it to configure memory easily.